src/containers/my-cooperations/cooperation-details/CooperationDetails.tsx

```diff
@@ -52,12 +52,25 @@ const CooperationDetails = () => {
   const { id } = useParams()
   const { isDesktop } = useBreakpoints()
   const { isActivityCreated } = useAppSelector(cooperationsSelector)
-  const [activeTab, setActiveTab] = useState<CooperationTabsEnum>(
-    CooperationTabsEnum.Activities
-  )
   const [isNotesOpen, setIsNotesOpen] = useState<boolean>(false)
   const [editMode, setEditMode] = useState<boolean>(false)
 
+  const [searchParams, setSearchParams] = useSearchParams()
+
+  const tab = searchParams.get('tab') as CooperationTabsEnum
```

Comment:
```suggestion
  const tab = searchParams.get('tab') ?? CooperationTabsEnum.Activities
```
Is it possible to add "default tab" by nullish coalescing operator here?

Reply:
I also need to check invalid tab values. When the user types something like `?tab=qwerty` it should also redirect to the default tab. That's why I prefer to keep the default tab as a separate concern.